frontend/src/models/types/RubricRating.ts

```diff
@@ -1,17 +1,24 @@
-import { Rating } from './types/rating.ts';
-import { v4 as uuid } from 'uuid';
+import { v4 as uuidv4 } from 'uuid';
+
+export interface RubricRating {
+  points: number;
+  description: string;
+  longDescription: string;
+  id?: number;
+  key: string; // UUID for React
+}
 
 export default function createRating(
   points: number = 0,
   description: string = '',
   longDescription: string = '',
   id: number | undefined = undefined
-): Rating {
+): RubricRating {
   return {
     points,
     description,
     longDescription,
     id,
-    key: uuid(),
```

Comment:
you really wanted the v4 there huh 😆

Reply:
Haha, I just realized that is only an alias 😆